docs/release/release_0_2_11.rst

```diff
@@ -0,0 +1,66 @@
+Announcement: napari 0.2.11
+===========================
+
+We're happy to announce the release of napari 0.2.11!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Highlights
+**********
+- Allow Points display properties to be set by point properties (#895)
```

Comment:
Here I would add a short paragraph to highlight this one, as it's a pretty major new API feature. e.g.,

> point face color and edge color are now settable as a property in a columnar data table, mapped using a colormap (continuous values) or a color cycle (categorical values). See `this example <https://github.com/napari/napari/blob/master/examples/add_points_with_properties.py>`_ for syntax details.

Reply:
btw markdown doesn't render ReST hyperlinks very well, here's the syntax reference fyi:

http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

Reply:
(**edit**: I've edited the above comment so that the syntax is properly shown)